src/styles/calendar/calendar.scss

```diff
@@ -0,0 +1,52 @@
+.calendar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 20px auto;
+  padding: 10px;
+  width: $calendar-width;
+  gap: $gap-size;
+  flex-wrap: wrap;
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    position: relative;
+    height: $day-size;
+    width: $day-size;
+    background-color: #eee;
+    border: 1px solid #000;
+    cursor: pointer;
+    transition: 0.5s;
```

Comment:
8'th point in checklist

Reply:
I don't quite understand what that means.

Reply:
it means `transition: background-color 0.5s transform 0.5s .... etc.`

Reply:
If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review